roles/kube-gateway/files/nginx.yaml

```diff
@@ -0,0 +1,10 @@
+init_config:
+
+instances:
+
+    -   nginx_status_url: https://localhost:7331/nginx_status/
```

Comment:
The listener as its configured below is not using TLS. The scheme here should be `http`, and you can drop the comments and the `ssl_validation` option below.